scripts/plugins/storage/sqlite/upgrade/15.sql

```diff
@@ -1,2 +1,4 @@
 CREATE INDEX IF NOT EXISTS log_ix2 ON log(ts);
 CREATE INDEX IF NOT EXISTS tasks_ix1 ON tasks(process_name, start_time);
+
+UPDATE foglamp.schedules SET enabled = 'f' WHERE schedule_name IN (SELECT key FROM foglamp.configuration WHERE json_extract(value, '$.plugin.value') = 'coap');
```

Comment:
Again, why are we disabling the schedule for COAP?

Reply:
A configured Coap south plugin might not be availabe in the new 1.4: better to disable the schedule.